code/mob/living/silicon/ai.dm

```diff
@@ -261,12 +261,14 @@ var/list/ai_emotions = list("Happy" = "ai_happy",\
 	SPAWN_DBG(0.6 SECONDS)
 		src.net_id = format_net_id("\ref[src]")
 
-		if(!src.link)
+		/*if(!src.link)
```

Comment:
delete it then

Reply:
Fixed; looks like I meant to remove it at some point and kept it just in case I needed to refer to it, but just forgot to